generators/server/templates/src/main/kotlin/package/service/KafkaConsumer.kt.ejs

```diff
@@ -0,0 +1,41 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.service
+
+import org.slf4j.LoggerFactory
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.stereotype.Service
+
+import java.io.IOException
+
+@Service
+class <%= upperFirstCamelCase(baseName) %>KafkaConsumer {
+
+    private val log = LoggerFactory.getLogger(<%= upperFirstCamelCase(baseName) %>KafkaConsumer::class.java)
+
+    @KafkaListener(topics = ["topic_<%= baseName.toLowerCase() %>"], groupId = "group_id")
+    @Throws(IOException::class)
+    fun consume(message: String) {
+        log.info("Consumed message in {} : {}", TOPIC, message)
```

Comment:
Would like to have interpolation here or create a ticket

Reply:
I was also thinking about that but the slf4j API does not actually perform the actual interpolation if the logging level is disabled, hence left it as is. That could be a small performance benefit. On the other hand, it is highly unlike for someone to disable the INFO level and interpolation is easier to read. BTW, there are more such occurrences in the code base. If we change it then we should keep it consistent. WDYT?